CONTRIBUTING.md

```diff
@@ -16,16 +16,16 @@ Go to https://github.com/argoproj/
 ## How to suggest a new feature
 
 Go to https://groups.google.com/forum/#!forum/argoproj
```

Comment:
Did you want to change URL to https://github.com/argoproj/argo/issues ?

Reply:
Thanks yes i meant to.